dev-php/phar-io-version/files/autoload.php

```diff
@@ -1,6 +1,30 @@
 <?php
```

Comment:
Same principle here for modifying a "known working" file.

Reply:
Fixed